src/components/RangeSlider/README.md

```diff
@@ -342,3 +342,46 @@ class RangeSliderExample extends React.Component {
 ## Related components
 
 - To collect a number value as a text input, [use the text field component](/components/forms/text-field)
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+The range slider provides a large click and tap target size for the slider thumbs. Merchants can also tap or click on the slider track to move the closest slider thumb.
```

Comment:
I don't have context on this, but does it change the meaning of the first sentence if we remove "size"?

Reply:
Nope! I can take it out.